rskj-core/src/test/java/co/rsk/peg/PegUtilsLegacyTest.java

```diff
@@ -578,11 +578,11 @@ void testIsValidPegInTx_hasChangeUtxoFromFlyoverErpFederation_afterRskip201_notP
 
         List<FederationMember> erpFedMembers = FederationTestUtils.getFederationMembersWithBtcKeys(erpFederationKeys);
         FederationArgs args = new FederationArgs(erpFedMembers, Instant.ofEpochMilli(1000L), 0L, networkParameters);
-        Federation erpFederation = FederationFactory.buildStandardMultiSigFederation(args);
```

Comment:
i dont know why the standard multisig fed was called erpFederation but i fixed it

Reply:
copy paste :)